frontend/src/scenes/instance/Licenses/index.tsx

```diff
@@ -8,63 +10,147 @@ import { PageHeader } from 'lib/components/PageHeader'
-                    placement="right"
-                    title="Maximum number of team members that you can have across all organizations with your current license."
-                >
-                    Max # of team members
-                    <InfoCircleOutlined className="info-indicator" />
-                </Tooltip>
-            )
+        {
+            title: 'Plan',
+            dataIndex: 'plan',
         },
-        render: function renderMaxUsers(license: any) {
-            return license.max_users === null ? 'Unlimited' : license.max_users
+        {
+            title: function Render() {
+                return (
+                    <Tooltip
+                        placement="right"
+                        title="Maximum number of team members that you can have across all organizations with your current license."
+                    >
+                        Max # of team members
+                        <InfoCircleOutlined className="info-indicator" />
+                    </Tooltip>
+                )
+            },
+            render: function renderMaxUsers(_, license: LicenseType) {
+                return license.max_users === null ? 'Unlimited' : license.max_users
+            },
         },
-    },
-    {
-        title: 'Key',
-        render: function renderActive(license: any) {
-            return <CodeSnippet>{license.key}</CodeSnippet>
+        {
+            title: 'Key',
+            render: function renderActive(_, license: LicenseType) {
+                return <CodeSnippet>{license.key}</CodeSnippet>
+            },
         },
-    },
-    {
-        title: 'License added on',
-        render: function renderActive(license: any) {
-            return humanFriendlyDetailedTime(license.created_at)
+        {
+            title: 'License added on',
+            render: function renderActive(_, license: LicenseType) {
+                return humanFriendlyDetailedTime(license.created_at)
+            },
         },
-    },
-]
+        {
+            width: 0,
+            render: function renderActive(_, license: LicenseType) {
+                let projectsToDelete = ((currentOrganization?.teams || []) as TeamType[])
+                    .filter((team) => !team.is_demo)
+                    .sort((a, b) => a.id - b.id)
+                projectsToDelete = projectsToDelete.slice(1, projectsToDelete.length)
 
-export function Licenses(): JSX.Element {
-    const [form] = Form.useForm()
-    const { licenses, licensesLoading, error } = useValues(licenseLogic)
-    const { createLicense } = useActions(licenseLogic)
+                return (
+                    <More
+                        overlay={
+                            <>
+                                <LemonButton
+                                    type="stealth"
+                                    style={{ color: 'var(--danger)' }}
+                                    onClick={() => {
+                                        Modal.confirm({
```

Comment:
This should be a `LemonModal` to match the design of our UI, probably in the style of `NewDashboardModal`. I put together a [quick sketch](https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=2567%3A1064) of what that'd look like:

| Ant | Lemon |
| --- | --- |
| <img width="416" alt="localhost_8000" src="https://user-images.githubusercontent.com/4550621/168094379-b1e1c5ca-28aa-4169-8e45-eaa2fcf8d31b.png"> | <img width="449" alt="modal - hierarchical" src="https://user-images.githubusercontent.com/4550621/168094383-80271d4c-fd26-4d15-9a9a-eba07c52a06c.png"> |